src/api/apps/articles/model/retrieve.js

```diff
@@ -0,0 +1,201 @@
+const _ = require("underscore")
```

Comment:
We are moving towards using `lodash` over `underscore` -- I believe most of the functions used here are available in in `lodash`, worth trying.

Reply:
Additionally for the imports you can use `import x from ...` rather than const.